conf/fail2ban/jail.local

```diff
@@ -14,30 +14,30 @@ action = %(action_mwl)s
 [ssh]
 maxretry = 7
 bantime = 3600
-# action = sendmail-whois-lines[name=ssh, dest="<USER>@<DOMAIN>.<TLD>,<USER2>@<DOMAIN2>.<TLD2>,fail2ban@blocklist.de", sender=fail2ban@box.<SERVER>.<TLD>, sendername="Fail2Ban"]
+action = sendmail-whois-lines[name=ssh, dest="admin@$DOMAIN", sender=fail2ban@$DOMAIN, sendername="Fail2Ban"]
```

Comment:
Is the intent on sending an email on every block? That can be achieved via a setting in the jail.local. This now no longer sends to blocklist.de. I don't think this is necessary.